sherpa/astro/datastack/tests/test_datastack.py

```diff
@@ -807,3 +807,19 @@ def test_operations_datastack_group(ds_setup, ds_datadir):
     assert np.allclose(d1.get_dep(filter=True)[15:20], [5., 5., 6., 7., 10.])
     datastack.ungroup('myid')
     assert np.all(d1.get_dep(filter=True)[15:20] == [3., 7., 1., 6., 4.])
+
+
+def test_wrapped_funcs_in_all():
+    """This is a regression test for #2270
+
+    Before #2270 several convenience functions using different
+    types of magic were used and interacted such that not every
+    wrapped function was added to __all__, in particular when one of them
+    is an alias for another. It's very unlikely that that same failure
+    mode will happen again, so this is a very rudamentary test where we are
+    just checking that both functions exisit.
```

Comment:
```suggestion
    Check that aliased routines (here set_model/set_source) are present
    both in the module and are exported by it. It is not worth checking every
    possible symbol.
```

I don't think we need the history.

We *could* make this a parametrized test (and test more symbols) by sending in the name to test, but - as you said in the now-deleted comment - it's not worth it.